go_installer.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 work_dir=$(echo $1 | sed 's:/*$::')
-version="1.22.4"
+version="1.23.1"
```

Comment:
You can also update the dependencies in go.mod file.

Reply:
Done